src/Trigger.ts

```diff
@@ -66,13 +76,20 @@ export default class Trigger {
   }
 
   public matches(
-    trigger: Trigger["trigger"],
-    options: Trigger["options"] | undefined,
-    target: Sprite | Stage
+    // TODO: Rework to not accept a symbol. Just compare to TriggerCreator.
```

Comment:
I wonder if we should split the implementation -- one `matches()` method that accepts a symbol and we can mark the entire method as deprecated (keep the same method name so as to not break old projects), and then make a new `matchesCreator()` method that does not have the deprecation warning.  That way we can smoothly warn old projects about how to upgrade, and provides a path for smooth migration?

Not super confident about this suggestion, so please take it with a grain of salt. :)